lighthouse-core/test/audits/byte-efficiency/offscreen-images-test.js

```diff
@@ -121,189 +106,208 @@ describe('OffscreenImages audit', () => {
+      endTime: 1.25,
+      priority: 'High',
+      timing: {receiveHeadersEnd: 1.25},
+    };
+    const recordB = {
+      url: 'https://example.com/b',
+      resourceSize: wastedSize,
+      requestId: 'b',
+      startTime: 1.25,
+      endTime: 1.5,
+      priority: 'High',
+      timing: {receiveHeadersEnd: 1.5},
+    };
+    const devtoolsLog = networkRecordsToDevtoolsLog([recordA, recordB]);
+
+    // Enough tasks to spread out graph.
+    const topLevelTasks = [
+      {ts: 1000, duration: 10},
+      {ts: 1050, duration: 10},
+      {ts: 1975, duration: 50},
+    ];
+    const artifacts = Object.assign(Runner.instantiateComputedArtifacts(), {
       ViewportDimensions: DEFAULT_DIMENSIONS,
       ImageUsage: [
         generateImage(generateSize(0, 0), [0, 0], recordA, recordA.url),
         generateImage(generateSize(200, 200), [3000, 0], recordB, recordB.url),
       ],
-      traces: {},
-      devtoolsLogs: {},
-      requestInteractive: async () => ({pessimisticEstimate: {nodeTimings: timings}}),
-      requestTraceOfTab: generateTraceOfTab(2),
-    }, [], context, [], context).then(auditResult => {
+      traces: {defaultPass: createTestTrace({topLevelTasks})},
+      devtoolsLogs: {defaultPass: devtoolsLog},
+    });
+
+    return UnusedImages.audit_(artifacts, [], context).then(auditResult => {
       assert.equal(auditResult.items.length, 2);
-      assert.equal(auditResult.items[0].url, 'a');
-      assert.equal(auditResult.items[1].url, 'b');
+      assert.equal(auditResult.items[0].url, recordA.url);
+      assert.equal(auditResult.items[0].wastedBytes, wastedSize);
+      assert.equal(auditResult.items[1].url, recordB.url);
+      assert.equal(auditResult.items[1].wastedBytes, wastedSize);
     });
   });
 
   it('rethrow error when interactive throws error in Lantern', async () => {
     context = {settings: {throttlingMethod: 'simulate'}};
-    try {
-      await UnusedImages.audit_({
-        ViewportDimensions: DEFAULT_DIMENSIONS,
-        ImageUsage: [
-          generateImage(generateSize(0, 0), [0, 0], generateRecord(100, 3), 'a'),
-          generateImage(generateSize(200, 200), [3000, 0], generateRecord(100, 4), 'b'),
-        ],
-        traces: {},
-        devtoolsLogs: {},
-        requestInteractive: generateInteractiveFuncError(),
-        requestTraceOfTab: generateTraceOfTab(2),
-      }, [], context, [], context);
-    } catch (err) {
-      return;
-    }
-    assert.ok(false);
-  });
+    const artifacts = Object.assign(Runner.instantiateComputedArtifacts(), {
+      ViewportDimensions: DEFAULT_DIMENSIONS,
+      ImageUsage: [
+        generateImage(generateSize(0, 0), [0, 0], generateRecord(100, 3), 'a'),
+        generateImage(generateSize(200, 200), [3000, 0], generateRecord(100, 4), 'b'),
+      ],
+      traces: {defaultPass: createTestTrace({traceEnd: 2000})},
+      devtoolsLogs: {},
+    });
 
-  it('finds images loaded before Trace End when interactive throws error (Lantern)', async () => {
```

Comment:
as far as I can tell this test was identical to the above. The start time of `generateRecord` of 'a' is the only thing that's different, but `throttlingMethod: 'simulate'` uses the network record timing information, not the timing info in the `ImageUsage` artifact.

Reply:
yeah I think this was very recently added when I had @exterkamp keep fiddling with the tests, probably got duped in the shuffle sorry!